Agent/Linux/ubuntu1804-README.md

```diff
@@ -0,0 +1,44 @@
+# Azure DevOps - Ubuntu 18.04 Agent
+
+The following sections provide a step-by-step guide to creating a Azure DevOps Agent Image using Packer, spin up an Agent using the Image and add it to the Azure DevOps Agent Pool.
+
+## Prepare your Azure DevOps Account
+
+1. Create an Azure DevOps Agent pool.
+
+2. Generate a Personal Access Token (PAT) for your Azure DevOps Organization. When generating the Personal Access Token (PAT), assign the following scopes:
+
+>a. Agent Pools - Read & Manage
+>b. Deployment Groups - Read & Manage
+
+3. Create a service principle and add it to your subscription as Owner. This is the subscription that will be used to host the image and Agent VM.
+
+4. Add the service principle to the subscription that will host the image and agent VMs with Owner role.
+
+## Build a Packer Image
+
+1. Build a Docker image and a container from the Docker image by mounting `Agent` folder as a volume in your Docker container.
+
+```
+docker run -it -v <path-to-vdc-folder>/Agent:/usr/app/src/Agent vdc:latest
+```
+
+2. Update the run.sh file with the service principle and subscription information.
+
+3. Run the command below from PowerShell terminal to build an VM Image
```

Comment:
a VM image, not an VM image. 

Add more information about what this image will be, and what it contains (i.e. No public ip, default nsg blocking ssh, etc. etc.)

Reply:
Image does not include the No public ip, default nsg blocking, etc..). Included this later in the Readme. Corrected the VM image grammatical error.